.github/workflows/measureMemory.yml

```diff
@@ -0,0 +1,412 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Measure memory
+
+# Controls when the action will run. Triggers the workflow push is only on main branch and PR on any branch.
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+  pull_request:
```

Comment:
Do you want to keep it for all pull requests?

Reply:
removed